src/resource/resource-opening-hours/ResourceOpeningHours.scss

```diff
@@ -1,76 +1,99 @@
 @import 'node_modules/hds-design-tokens/lib/all.scss';
 
-.resource-opening-hours-header {
+.opening-periods-header {
+  --background-color: var(--opening-periods-header-background-color);
+  --font-color: var(--opening-periods-header-color);
+  --periods-header-button-border-color: var(--opening-periods-header-button-border-color);
+  --periods-header-button-color: var(--opening-periods-header-button-color);
+  --periods-header-button-focus-color: var(--opening-periods-header-button-focus-color);
+  --periods-header-button-focus-border-color: var(--opening-periods-header-button-focus-border-color);
+  --periods-header-button-hover-background-color: var(--opening-periods-header-button-hover-background-color);
+  --periods-header-button-hover-color: var(--opening-periods-header-button-hover-color);
+
   min-height: 4rem;
-  background-color: var(--resource-opening-hours-header-background-color);
+  background-color: var(--background-color);
+  color: var(--font-color);
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
   margin-top: auto;
   padding-left: $spacing-m;
   padding-right: $spacing-m;
+
+  .opening-period-header-button {
+    font-family: HelsinkiGrotesk-Medium, Arial, sans-serif;
+    color: var(--periods-header-button-color);
+    border-color: var(--periods-header-button-border-color) !important;
+
+    &:hover {
+      background-color: var(--periods-header-button-hover-background-color) !important;
+      color: var(--periods-header-button-hover-color) !important;
+    }
+
+    &:focus {
+      color: var(--periods-header-button-focus-color);
+      border-color: var(--periods-header-button-focus-border-color) !important;
+      border-width: 3px;
+    }
+  }
+}
+
+.opening-periods-header-light {
+  --background-color: var(--opening-periods-header-light-background-color);
```

Comment:
Where are these variables used?

Reply:
This overrides previously set CSS-variables. The theming is made by using those. I could relocate this declaration to make it more readable 🤔

Reply:
How about refactoring this so that there's one common class, which consists things that do not change. And then separate classes for the themes, which define background-colors and these other settings?

Reply:
I can try to change it if it is more readable.